base/db/R/query.dplyr.R

```diff
@@ -4,19 +4,32 @@
 #' 
 betyConnect <- function(php.config = "../../web/config.php") {
   ## Read PHP config file for webserver
+  if (file.exists(php.config)) {
+    php_params <- PEcAn.utils::read_web_config(php.config)
+  } else {
+    php_params <- list()
+  }
+
+  ## helper function
+  getphp = function (item, default = "") {
+    value = php_params[[item]]
+    if (is.null(value)) default else value
+  }
+
+  ## fill in all data from environment variables
+  dbparams <- get_postgres_envvars(host = getphp("db_bety_hostname", "localhost"),
+                                   port = getphp("db_bety_port", "5432"),
+                                   dbname = getphp("db_bety_database", "bety"),
+                                   user = getphp("db_bety_username", "bety"),
+                                   password = getphp("db_bety_password", "bety"))
 
-  config.list <- PEcAn.utils::read_web_config(php.config)
+  ## force driver to be postgres, needed when switching to db,open
```

Comment:
```suggestion
  ## force driver to be postgres (only value supported by db.open)
```

(for the benefit of folks reading later who don't know the commit history)